docs/user/introduction/features.rst

```diff
@@ -75,6 +75,19 @@ shows a live calculation of masternode earnings. These rewards decrease
 by 7% each year, together with the block reward. There is also the
 possibility for masternodes to earn money from fees in the future.
 
+High-Performance Masternodes (HPMNs)
+
+High-Performance Masternodes (HPMNs) are a subset of Masternodes that have 
+been created to host the Dash Platform. An HPMN is a lot like Masternodes 
+with three main differences:
+
```

Comment:
Looking at the section as a whole, I think adding a table would help easily distinguish between regular and high-performance MNs. We could have a line for collateral requirement, governance votes, and services provided (Core-only / Core and Platform).

| | Regular | High-performance |
|-|-|-|
| Required Collateral | 1000 DASH | 4000 DASH |

...

Other references in this section to collateral and votes should be updated and could reference this table